### PR TITLE
Kill some more plugins

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15545)

### CODE REVIEW

**Comment by SimonSapin** [comment]
r=me with or without the renames, as you feel like.

---

Reviewed 4 of 4 files at r1, 16 of 16 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[components/privatize_derive/Cargo.toml, line 2 at r2](https://reviewable.io:443/reviews/servo/servo/15545#-Kcxe7I8S_gPhbzdhYoQ:-Kcxe7I8S_gPhbzdhYoR:b2jtgqz) ([raw file](https://github.com/servo/servo/blob/1fca675e4addee60ad59292758e7b98176ef46e4/components/privatize_derive/Cargo.toml#L2)):*
> ```TOML
> [package]
> name = "privatize_derive"
> ```

Super nit: call this `deny_public_fields` to align with the derive name?

---

*[components/privatize_derive/lib.rs, line 9 at r2](https://reviewable.io:443/reviews/servo/servo/15545#-KcxeK6ssRUlTnLXPKkZ:-KcxeK6ssRUlTnLXPKk_:b-pix5yg) ([raw file](https://github.com/servo/servo/blob/1fca675e4addee60ad59292758e7b98176ef46e4/components/privatize_derive/lib.rs#L9)):*
> ```Rust
> extern crate synstructure;
> 
> #[proc_macro_derive(deny_public_fields)]
> ```

Super nit: `DenyPublicFields`?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/servo/15545#-:-Kcxfsn-fmajmHTvqNRZ:bjodbl1)*

**Comment by Ms2ger** [comment]
```
The command "./mach test-compiletest" exited with 101.
```

File: components/script/dom/bindings/iterable.rs [hunk 1]
```diff
@@ -47,40 +47,14 @@ pub trait Iterable {
...
-#[derive(HeapSizeOf)]
+#[dom_struct]
 pub struct IterableIterator<T: DomObject + JSTraceable + Iterable> {
     reflector: Reflector,
     iterable: JS<T>,
     type_: IteratorType,
     index: Cell<u32>,
 }
 
-impl<T: DomObject + JSTraceable + Iterable> DomObject for IterableIterator<T> {
```
Comment (Manishearth) [inline components/script/dom/bindings/iterable.rs]: Why did this need to be removed?

**Review by Manishearth — APPROVED** [review]
r=me

File: components/script/dom/bindings/iterable.rs (hunk 1 above)
Comment (SimonSapin) [inline components/script/dom/bindings/iterable.rs]: `#[dom_struct]` adds `#[derive(DomObject)]`, which provides this impl.

**Pull request merged**